index.html

```diff
@@ -5,36 +5,17 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Calendar to search at GitHub</title>
+  <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
   <link rel="stylesheet" href="src/css/style.css">
 </head>
 <body>
   <main id="app">
-    <form id="calendary_filter" class="calendary_filter">
-        <select id="type_of_search" name="typeOfSearch" class="input">
-          <option value="issue">
-            Issues
-          </option>
-          <option value="pr">
-            Pull Request
-          </option>
-        </select>
-        <input 
-          type="text" 
-          placeholder="User/Organization" 
-          list="users_history"
-          id="input_user" 
-          name="user" 
-          class="input calendary_filter__repository_name" 
-          autofocus
-        />
-        <input 
-          type="text" 
-          placeholder="Repository name" 
-          list="repositories_history"
-          id="input_repository" 
-          name="repository" 
-          class="input calendary_filter__repository_name" 
-        />
```

Comment:
I would suggest hiding this elements using CSS since this elements will be used for the next part of the form that will develop in #4

Reply:
I think there's something wrong, this class is never used (btn--disabled). The button with no color has the class (.btn.) On the other hand, the inputs that are never used were erased since the first commit. Anyway, I will check and try to figure out what happend. I will leave the actual view of the last commit, for your consideration.
![Sin título](https://user-images.githubusercontent.com/92061563/157159517-b5c28364-b568-4fee-8d8c-31dd439f5c68.png)

Reply:
Hola buenas!. A lo que me refería es que esos inputs aunque no se utilizan en esta vista, sí se utilizan más adelante, por eso recomiendo "ocultarlos" o "comentarlos" en lugar de borrarlos, ya que se ahorra trabajo para developer que trabaje en esa otra parte, porque la lógica de guardar los datos y etc ya está funcionando con esos inputs tal cual están, solamente hace falta crear una lógica de formulario en 2 pasos.

Si te fijas en Figma, ese formulario tiene 2 partes:
Parte 1: Usuario selecciona el usuario, issues y repositorio
![imagen](https://user-images.githubusercontent.com/66505715/157272861-24126aae-aed2-448e-ba43-d74e858c61f9.png)

Parte 2: El usuario selecciona la fecha, y si quiere ver issues / repositorios Después de esa fecha, antes de esa fecha o en esa misma fecha. Esta es la vista que tú estás trabajando
![imagen](https://user-images.githubusercontent.com/66505715/157272963-52a8a2c5-dbcf-4a79-8725-31369b3fd75b.png)

Entonces por eso propongo mejor ocultar o comentar esos inputs, ya que de todas formas el siguiente developer que trabaje esa parte necesitaría añadir esos mismos inputs

https://www.figma.com/file/Pw5EfnvCLnXox2mFS40kdW/Azordev?node-id=113%3A144

-----------------------------------------------------------------------------------------------------------------------------

En cuanto a la clase "btn--disabled" es una clase que se coloca con JavaScript al desactivar un btn, seguro se puede mejorar esa lógica! Yo lo hice porque se usaba un botón y un elemento de link, y con el elemento de link no funcionaba la pseudoclase a::disabled. Pero claro que se puede mejorar

![imagen](https://user-images.githubusercontent.com/66505715/157272162-67711b4e-163a-4594-8471-57c969e5bb2f.png)

Reply:
Please see latest changes. Leave a comment.